### ISSUE TITLE
Multiple label definition

### ISSUE THREAD

**Issue by GiulioRomualdi** [orsharir/github-mathjax#28]
Hi @orsharir,  I have an issue when I add the command `\label{}` to the environment _equation_.
In details when I create an equation as follows, I retrive the error `Label 'eq:test' multiply defined`.
```latex
\begin{equation}
\label{eq:test}
sin^2(x) + cos^2(x) = 1
\end{equation}
```
\begin{equation}
\label{eq:test}
sin^2(x) + cos^2(x) = 1
\end{equation}

Unfortunately, I notice that the problem happens when the equation is cited by using the command `\eqref{}` e.g. \eqref{eq:test}
Do you know why this is happening? Do you have any suggestions to fix the problem?

**Comment by muzimuzhi** [comment]
I can re-produce your problem when referring to any label by `\ref` or `\eqref`.

This may relates to the following "feature" of MathJax

> If you are using automatic equation numbering and modifying the page dynamically, you can run into problems due to duplicate labels. See [Reset Automatic Equation Numbering](http://docs.mathjax.org/en/latest/advanced/typeset.html#reset-equation-numbers) for how to address this.
> 
> ref: http://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering
